karb/parser.js

```diff
@@ -74,7 +85,11 @@ module.exports = new Parser(function analyseEC(parsedUrl, ec) {
     // /document/kli-ka-ai-2020-01-002?title=Arbitration%20International
     // /document/KLI-KCC-1103103-n
 
-    result.rtype  = param.title ? 'ARTICLE' : 'BOOK_SECTION';
+    if (param.title && param.title != 'PDF') {
```

Comment:
Prefer to use a strict inequality for string comparison. ``param.title !== 'PDF'``